### ISSUE TITLE
please implement a responsiveness in the screen

### ISSUE THREAD

**Issue by aashishshrestha5532** [byprogrammers/lets-code-react-native#1]
In my lowest resolution emulator i have observed that the design doesnot look responsive

Main Screen
![mobiles1](https://user-images.githubusercontent.com/26792137/91700843-2ebd4880-eb96-11ea-8354-8d95d367a04f.png)

In the home screen the texts are overlapped.

Detail Screen
![mobiles2](https://user-images.githubusercontent.com/26792137/91700848-2fee7580-eb96-11ea-9b50-1c6d15e671c2.png)

In the second screen the content(About contents) is clipped.Should have use scrollview for that.

**Comment by jacktiong92** [comment]
Yeah I think you are right. Using A scrollview might be a better option in this case. I will change it once I'm able to or you can submit a pull request as well
